src/hll.c

```diff
@@ -585,6 +585,31 @@ typedef struct
 
 } bitstream_read_cursor_t;
 
+
+/*
+ * The result of mse_nelem_max()/msc_regs_idx_limit() is a const in a specific platform.
+ * But we couldn't hard code it explicitly because we do not know current alignment schema
+ * when compiling it.
+ */
+static size_t
```

Comment:
These functions should have prototypes declared at top of file with other static prototypes

Reply:
seems there are no declarations for most other functions